packages/binding-mqtt/README.md

```diff
@@ -31,7 +31,7 @@ MqttBrokerServer = require("@node-wot/binding-mqtt").MqttBrokerServer;
 
 // create Servient add MQTT binding
 let servient = new Servient();
-servient.addServer(new MqttBrokerServer({uri:"mqtt://test.mosquitto.org"}));
+servient.addServer(new MqttBrokerServer({ uri: "mqtt://test.mosquitto.org" }));
```

Comment:
I don't think this change belongs to this PR ... 
It is not a big deal if we leave it in but somewhat confusing

Reply:
I can prepare another PR for it. I was using it for testing :)